src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasPointAndLineSlicer.java

```diff
@@ -308,10 +308,17 @@ private Geometry cutGeometry(final long identifier,
                         ISOCountryTag.KEY);
                 CountryBoundaryMap.setGeometryProperty(clipped, ISOCountryTag.KEY, countryCode);
                 addResult(clipped, results);
-
-                // Update target to be what's left after clipping
-                currentTarget = currentTarget.difference(candidate);
-
+                try
+                {
+                    currentTarget = currentTarget.difference(candidate);
+                }
+                catch (final IllegalArgumentException exception)
+                {
+                    logger.warn(
```

Comment:
If that usually means there is funky data in OSM, should we make that an ERROR? Or is that purely JTS?

Reply:
in this case, it's really just an extremely rare possibility of good data for this particular geometry operation, so I think WARN is appropriate.